proto/minder/v1/minder.proto

```diff
@@ -1004,8 +996,7 @@ message Provider {
 // Context defines the context in which a rule is evaluated.
 // this normally refers to a combination of the provider, organization and project.
 message Context {
-    string provider = 1;
-    optional string organization = 2;
```

Comment:
Can you rename this to retired_organization and put at the bottom of the message, so we don't re-use id 2?

Reply:
1. So the practice of deprecating a field is to prefix it? 
2. Is it the same if it's not optional?
3. And last one :D Is it accepted to remove the prefix if at some point we decide to re-introduce that label?

Reply:
Thanks @evankanderson, addressed in https://github.com/stacklok/minder/pull/1831/commits/907044c61f3ea534531537de81cbf6254498652c.